application/run.sh

```diff
@@ -21,20 +18,22 @@ chown -R www-data:www-data \
 echo 'sleeping a random number of seconds up to 9 to avoid migration runs clash'
 sleep $[ ( $RANDOM % 10 ) ]s
 
+if [[ -z "${APP_ID}" ]]; then
+  /data/yii migrate --interactive=0
 
-# Run database migrations
-/data/yii migrate --interactive=0
-
-if [[ ! -z $RUN_TASK ]]; then
+  if [[ ! -z $RUN_TASK ]]; then
     ./yii $RUN_TASK
     exit $?
-fi
+  fi
 
-apache2ctl -k start -D FOREGROUND
+  apache2ctl -k start -D FOREGROUND
+else
+  config-shim -v --app $APP_ID --config $CONFIG_ID --env $ENV_ID /data/yii migrate --interactive=0
```

Comment:
Is there a way to have config-shim add the data to the existing environment, so that we don't have to prefix every command with the call to config-shim?

Reply:
Not directly. It's not possible to change the environment of a parent process (e.g. bash) but it might be possible to write it to a file and then read from that file on each script line that needs it. I chose not to go down that road because it's more complex and because I would like to simplify this script in other ways such that it would only require one call to `config-shim`. What do you think?

Reply:
Good call 👍